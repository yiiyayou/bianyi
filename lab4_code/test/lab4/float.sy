int   ee=10;
int   ff=0750123;
int   gg=0x12f;
float me=1.732;
float aa=0.123;
float bb=123E2;
float cc=12.398e4;
float dd=12.398E-6;
float hh=0x1.921fb6p+1;
float ii=0x1.AP-3;
float jj=075.17p-3;