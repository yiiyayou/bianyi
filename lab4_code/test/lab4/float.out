[DEBUG LAB4]: 	1		0		INT			int		
[DEBUG LAB4]: 	1		6		ID			ee			0x55fcfd01b5a0
[DEBUG LAB4]: 	1		8		ASSIGN		=		
[DEBUG LAB4]: 	1		9		NUMBER		10		
[DEBUG LAB4]: 	1		9		SEMI		;		
[DEBUG LAB4]: 	2		0		INT			int		
[DEBUG LAB4]: 	2		6		ID			ff			0x55fcfd01c600
[DEBUG LAB4]: 	2		8		ASSIGN		=		
[DEBUG LAB4]: 	2		9		NUMBER		249939		
[DEBUG LAB4]: 	2		9		SEMI		;		
[DEBUG LAB4]: 	3		0		INT			int		
[DEBUG LAB4]: 	3		6		ID			gg			0x55fcfd01c650
[DEBUG LAB4]: 	3		8		ASSIGN		=		
[DEBUG LAB4]: 	3		9		NUMBER		303		
[DEBUG LAB4]: 	3		9		SEMI		;		
[DEBUG LAB4]: 	4		0		FLOAT		float	
[DEBUG LAB4]: 	4		6		ID			me			0x55fcfd01c6d0
[DEBUG LAB4]: 	4		8		ASSIGN		=		
[DEBUG LAB4]: 	4		9		NUMBER		1.732000		
[DEBUG LAB4]: 	4		9		SEMI		;		
[DEBUG LAB4]: 	5		0		FLOAT		float	
[DEBUG LAB4]: 	5		6		ID			aa			0x55fcfd01c720
[DEBUG LAB4]: 	5		8		ASSIGN		=		
[DEBUG LAB4]: 	5		9		NUMBER		0.123000		
[DEBUG LAB4]: 	5		9		SEMI		;		
[DEBUG LAB4]: 	6		0		FLOAT		float	
[DEBUG LAB4]: 	6		6		ID			bb			0x55fcfd01c770
[DEBUG LAB4]: 	6		8		ASSIGN		=		
[DEBUG LAB4]: 	6		9		NUMBER		12300.000000		
[DEBUG LAB4]: 	6		9		SEMI		;		
[DEBUG LAB4]: 	7		0		FLOAT		float	
[DEBUG LAB4]: 	7		6		ID			cc			0x55fcfd01c7c0
[DEBUG LAB4]: 	7		8		ASSIGN		=		
[DEBUG LAB4]: 	7		9		NUMBER		123980.000000		
[DEBUG LAB4]: 	7		9		SEMI		;		
[DEBUG LAB4]: 	8		0		FLOAT		float	
[DEBUG LAB4]: 	8		6		ID			dd			0x55fcfd01c810
[DEBUG LAB4]: 	8		8		ASSIGN		=		
[DEBUG LAB4]: 	8		9		NUMBER		0.000012		
[DEBUG LAB4]: 	8		9		SEMI		;		
[DEBUG LAB4]: 	9		0		FLOAT		float	
[DEBUG LAB4]: 	9		6		ID			hh			0x55fcfd01c860
[DEBUG LAB4]: 	9		8		ASSIGN		=		
[DEBUG LAB4]: 	9		9		NUMBER		3.141593		
[DEBUG LAB4]: 	9		9		SEMI		;		
[DEBUG LAB4]: 	10		0		FLOAT		float	
[DEBUG LAB4]: 	10		6		ID			ii			0x55fcfd01c8b0
[DEBUG LAB4]: 	10		8		ASSIGN		=		
[DEBUG LAB4]: 	10		9		NUMBER		0.203125		
[DEBUG LAB4]: 	10		9		SEMI		;		
[DEBUG LAB4]: 	11		0		FLOAT		float	
[DEBUG LAB4]: 	11		6		ID			jj			0x55fcfd01c900
[DEBUG LAB4]: 	11		8		ASSIGN		=		
[DEBUG LAB4]: 	11		9		NUMBER		75.169998		
[DEBUG LAB4]: 	11		9		SEMI		;		
